cms/djangoapps/contentstore/views/tests/test_block.py

```diff
@@ -786,6 +786,29 @@ def verify_name(source_usage_key, parent_usage_key, expected_name, display_name=
         # Now send a custom display name for the duplicate.
         verify_name(self.seq_usage_key, self.chapter_usage_key, "customized name", display_name="customized name")
 
+    def test_shallow_duplicate(self):
+        """
+        Test that shallow_duplicate creates a new block.
+        """
+        source_course = CourseFactory()
+        user = UserFactory.create()
+        source_chapter = BlockFactory(parent=source_course, category='chapter', display_name='Source Chapter')
+        BlockFactory(parent=source_chapter, category='html', display_name='Child')
+        # Refresh.
+        source_chapter = self.store.get_item(source_chapter.location)
+        self.assertEqual(len(source_chapter.get_children()), 1)
+        destination_course = CourseFactory()
+        destination_location = duplicate_block(
+            parent_usage_key=destination_course.location, duplicate_source_usage_key=source_chapter.location,
```

Comment:
```suggestion
            parent_usage_key=destination_course.location,
            duplicate_source_usage_key=source_chapter.location,
```
nit - should be consistently one argument per line.